docs/v6-v7-upgrade.md

```diff
@@ -0,0 +1,17 @@
+# Upgrading from V6 to V7
+
+Most changes are relatively minor and shouldn't affect most users.
+
+# Changes
+
+## `throws` option now rejects a Promise
+A regression was introduced in v6 whereby the `throws: true` option woudl throw an uncaught error. The `fetch` api catches all its internal errors and returns a rejected `Promise` in every case, so this change has been reverted to be more useful for mocking typical `fetch` errors.
+
+## Responses are wrapped in an ES Proxy
+This is to enable a more powerful `flush()` method, able to wait for asynchronous resolution of response methods such as `.json()` and `.text()`. `flush(true)` will resolve only when Promises returnes by any response methods called before `flush()` have resolved
+
+## Supports resolving dots in urls
```

Comment:
@adam-nielsen Would welcome your comments on these docs.

After this is merged into the v7 branch I'll release another alpha so you can use it